hoomd/hpmc/ComputeSDF.h

```diff
@@ -264,27 +269,64 @@ template<class Shape> void ComputeSDF<Shape>::zeroHistogram()
     // Zero the histogram
     for (size_t i = 0; i < m_hist.size(); i++)
         {
-        m_hist[i] = 0;
+        m_hist[i] = 0.0;
         }
     }
 
+template<class Shape> Scalar ComputeSDF<Shape>::getMaxInteractionDiameter()
+    {
+    Scalar max_core_diameter = m_mc->getMaxCoreDiameter();
+    Scalar max_r_cut_patch(0.0);
+
+    if (m_mc->getPatchEnergy())
+        {
+        const std::vector<param_type, hoomd::detail::managed_allocator<param_type>>& params
+            = m_mc->getParams();
```

Comment:
```suggestion
        const auto& params = m_mc->getParams();
```
Let the compiler do the work.